src/library_glfw.js

```diff
@@ -1055,6 +1060,13 @@ var LibraryGLFW = {
       }
     },
 
+    defaultWindowHints: () => {
+      GLFW.hints = {};
+      for (var k in GLFW.defaultHints) {
+        GLFW.hints[k] = GLFW.defaultHints[k];
+      }
+    },
```

Comment:
Looks like this can be rebased to make this change smaller now that https://github.com/emscripten-core/emscripten/pull/20770 has landed?

Reply:
I am in the process of writing details about this changes.. just hold on a secondf on the review

Reply:
code has been rebased and this is gone